trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.post.test.js

```diff
@@ -0,0 +1,149 @@
+import { isTfmFacilityEndDateFeatureFlagEnabled, MAPPED_FACILITY_TYPE, TEAM_IDS } from '@ukef/dtfs2-common';
+import api from '../../../api';
+import { mockRes } from '../../../test-mocks';
+import { MOCK_AMENDMENT_COVERENDDATE_CHANGE, MOCK_AMENDMENT_FACILITYVALUE_AND_COVERENDDATE_CHANGE } from '../../../test-mocks/amendment-test-mocks';
```

Comment:
Should these be the following?:

```js
MOCK_AMENDMENT_COVER_END_DATE_CHANGE
MOCK_AMENDMENT_FACILITY_VALUE_AND_COVER_END_DATE_CHANGE
```

Reply:
If this is an existing thing, not a blocker for this PR

Reply:
This is an existing pattern from at least 2 years ago so we've stuck to it for now. I agree it should be reviewed when we can. My interpretation is it probably allows easier reading of multi word constants but that may or may not be correct